go/service/chat_local.go

```diff
@@ -6,7 +6,9 @@ package service
 import (
 	"encoding/hex"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"math/rand"
```

Comment:
See `NewDeviceID` for how to make a new ID.

Also, we should reserve a trailing two-byte suffix for this purpose, as we do elsewhere.